AppDevAnnouncements/Classes/NotificationViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  NotificationViewController.swift
+//  AppDevAnnouncements
+//
+//  Created by Gonzalo Gonzalez on 3/11/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol NotificationDelegate: class {
+    func present(_ notificationViewController: NotificationViewController)
+}
+
+class NotificationViewController: UIViewController {
+
+    var notificationView: NotificationView!
+
+    var visual : String?
+    var subject : String?
+    var body : String?
+    var ctaText : String?
+    var ctaAction : String?
+
+    var presenter: UIViewController?
+    weak var delegate : NotificationDelegate?
+
+    convenience init(visual: String?, subject: String?, body: String?, ctaText: String?, ctaAction: String?, delegate: NotificationDelegate) {
```

Comment:
Do we have an idea of what patterns of information we might pass into the NotificationViewController? Like do we want everything to be optional? Maybe we can have more than just one initializer for certain common cases.

Reply:
Also is delegate the only non-optional?? If so, make the weak var stored in the VC non-optional as well

Reply:
I decided to make everything non-optional so the weak var is just a regular non-optional now @Omarrasheed. I don't know what cases there could be yet so for now I'll just assume all fields are definitely provided and keep the one initializer @lucyxubroad.